apps/etterlatte-saksbehandling-ui/client/src/components/person/journalfoeringsoppgave/nybehandling/FullfoerOppgaveModal.tsx

```diff
@@ -56,7 +64,9 @@ export default function FullfoerOppgaveModal({ oppgave, behandlingBehov }: Modal
             ferdigstilles.
           </BodyLong>
 
-          {isSuccess(opprettBehandlingStatus) && isSuccess(ferdigstillOppgaveStatus) ? (
+          {isSuccess(opprettBehandlingStatus) &&
```

Comment:
Litt pirk, men kanskje vi skulle hatt slike metoder som tar inn en liste med argumenter

Reply:
Ja, det hadde vært ideelt, men litt usikker på hvordan vi skulle løst det i praksis. Sett at hver enkelt skal ha egen tekst for feil/lasting. 🤔